src/env.js

```diff
@@ -70,6 +75,16 @@ const envDefinitions = {
     desc:
       "force implementation for ALL currency bridges (affects scanning accounts)",
   },
+  COSMOS_GAS_AMPLIFIER: {
+    def: 2,
+    parser: intParser,
+    desc: "estimate gas multiplier",
+  },
+  COSMOS_GAS_PRICE: {
+    def: 0.025,
+    parser: floatParser,
+    desc: "gasLimit * gasPrice to determine the fees price",
```

Comment:
"A too low GAS_PRICE will get rejected before the transaction is broadcast" _may_ be added